apps/pigment-next-app/README.md

```diff
@@ -13,6 +13,8 @@ pnpm build:zero
 Then start the Next.js development server:
 
 ```bash
+cd apps/
```

Comment:
```suggestion
cd apps/pigment-next-app
```
You can `cd` to the app, `pnpm install` still work the same.

Reply:
Seems like this is handled separately in https://github.com/mui/material-ui/pull/41393, I will revert these changes.